metals/src/main/scala/scala/meta/internal/decorations/DecorationClient.scala

```diff
@@ -0,0 +1,34 @@
+package scala.meta.internal.decorations
+
+import javax.annotation.Nullable
+import org.eclipse.lsp4j.jsonrpc.services.JsonNotification
+import org.eclipse.lsp4j.Range
+import org.eclipse.lsp4j.MarkedString
+
+trait DecorationClient {
+  @JsonNotification("metals/decorationTypeDidChange")
+  def metalsDecorationTypeDidChange(
+      params: DecorationTypeDidChange
+  ): Unit
+  @JsonNotification("metals/publishDecorations")
+  def metalsDecorationRangesDidChange(
```

Comment:
Good catch, the name was originally `metals/decorationRangesDidChange` but then I changed to `metals/publishDecorations` since the API feels closer to `textDocument/publishDiagnostics` than `workspace/didChangeConfiguration`